### PR TITLE
register US

### PR DESCRIPTION
**Pull request by Wowlfy**

### CODE REVIEW

**Review by maxime-guichard — APPROVED** [review]

File: migrations/Version20210625105228.php [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+    public function down(Schema $schema): void
+    {
+        // this down() migration is auto-generated, please modify it to your needs
+        $this->addSql('ALTER TABLE job DROP FOREIGN KEY FK_FBD8E0F8E7BE1239');
+        $this->addSql('DROP INDEX IDX_FBD8E0F8E7BE1239 ON job');
+        $this->addSql('ALTER TABLE user DROP FOREIGN KEY FK_8D93D649979B1AD6');
+        $this->addSql('DROP INDEX UNIQ_8D93D649979B1AD6 ON user');
+        $this->addSql('ALTER TABLE user DROP company_id, DROP phone');
+    }
+}
```
Comment (LucyMerlier) [inline migrations/Version20210625105228.php:39]: there is already a migration that does that on dev

File: src/Entity/User.php [hunk 2]
```diff
@@ -4,10 +4,12 @@
 
 use App\Repository\UserRepository;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Security\Core\User\UserInterface;
 
 /**
  * @ORM\Entity(repositoryClass=UserRepository::class)
+ * @UniqueEntity(fields={"email"}, message="There is already an account with this email")
```
Comment (LucyMerlier) [inline src/Entity/User.php:12]: error message should be in french

File: src/Form/RegistrationFormType.php [hunk 3]
```diff
@@ -0,0 +1,64 @@
+<?php
+
+namespace App\Form;
+
+use App\Entity\User;
+use phpDocumentor\Reflection\Types\Null_;
+use PHPMD\Utility\Strings;
```
Comment (LucyMerlier) [inline src/Form/RegistrationFormType.php:7]: what is this for?

File: src/Form/RegistrationFormType.php [hunk 4]
```diff
@@ -0,0 +1,64 @@
...
+use Symfony\Component\BrowserKit\Response;
```
Comment (LucyMerlier) [inline src/Form/RegistrationFormType.php:8]: what is this for?

File: src/Form/RegistrationFormType.php [hunk 5]
```diff
@@ -0,0 +1,64 @@
...
+use Symfony\Component\Form\Extension\Core\Type\EmailType;
+use Symfony\Component\Form\Extension\Core\Type\PasswordType;
+use Symfony\Component\Form\Extension\Core\Type\TelType;
+use Symfony\Component\Form\Extension\Core\Type\TextType;
+use Symfony\Component\Form\FormBuilderInterface;
+use Symfony\Component\OptionsResolver\OptionsResolver;
+use Symfony\Component\Validator\Constraints\IsTrue;
+use Symfony\Component\Validator\Constraints\Length;
+use Symfony\Component\Validator\Constraints\NotBlank;
+use Symfony\Flex\Response as FlexResponse;
```
Comment (LucyMerlier) [inline src/Form/RegistrationFormType.php:20]: what is this for?

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments

**Review by maxime-guichard — DISMISSED** [review]

File: src/Entity/User.php [hunk 6]
```diff
@@ -4,10 +4,12 @@
 
 use App\Repository\UserRepository;
 use Doctrine\ORM\Mapping as ORM;
+use Symfony\Bridge\Doctrine\Validator\Constraints\UniqueEntity;
 use Symfony\Component\Security\Core\User\UserInterface;
 
 /**
  * @ORM\Entity(repositoryClass=UserRepository::class)
+ * @UniqueEntity(fields={"email"}, message="Cette Email à deja été utilisé sur ce site.")
```
Comment (LucyMerlier) [inline src/Entity/User.php:12]: cet*

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comment

**Review by maxime-guichard — APPROVED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**